contracts/test/strategies/fraxeth.js

```diff
@@ -40,7 +40,7 @@ describe("FraxETH Strategy", function () {
   }));
 
   describe("Mint", function () {
-    it("Should allow minting with frxETH", async () => {
+    it.skip("Should allow minting with frxETH", async () => {
```

Comment:
I know we will have LSTs in the vault for now. But I think we can either: 
- delete the LST mint tests
- change the test so that revert of "unsupported asset" is expected

Reply:
The fork tests ensure that it reverts. Can update the unit tests as well